plugins/connection/aws_ssm.py

```diff
@@ -82,6 +82,10 @@
     version_added: 2.2.0
     vars:
     - name: ansible_aws_ssm_bucket_sse_kms_key_id
+  ssm_document:
+    description: SSM document to use when connecting.
+    vars:
+    - name: ansible_aws_ssm_document
```

Comment:
@blinkseb Thank you for taking time to work on this feature idea. Please add `version_added: 3.1.0`

Reply:
done :)